lib/test/clojure_lsp/features/code_lens_test.clj

```diff
@@ -114,4 +115,44 @@
                (h/file-uri "file:///a.clj") 1 13
                {:start {:line 1 :character 5}
                 :end {:line 1 :character 12}}
-               (h/db)))))))
+               (h/db))))))
+  (testing "test references"
```

Comment:
I couldn't find unit tests for the splitting of test and main code-lens references, so this adds them.

Reply:
thanks!